CONTRIBUTING.md

```diff
@@ -48,7 +56,7 @@ Status of closed issues:
 -   works: The issue cannot be reproduced, or the feature is working as expected.
 -   wontfix: The issue will not be fixed.
 
```

Comment:
author action : The person who reported the issue needs to provide additional information.

Reply:
The content you highlighed is original content that I kept there.